deployments/a11y/image/environment.yml

```diff
@@ -0,0 +1,91 @@
+name: a11y
+
+channels:
+- conda-forge
+
+dependencies:
+- python==3.9
+- syncthing==1.20.4
+- git
+#- altair==4.2.0
+#- beautifulsoup4==4.11.1
+#- black==22.6.0
+#- bokeh==2.4.3
+- bqplot==0.12.34
+#- cartopy==0.21.0
+- coverage==6.4.4
+#- cython==0.29.32
+#- dask==2022.8.1
+#- dask-labextension==5.3.0
+#- fortran-magic==0.7
+#- h5netcdf==1.0.2
+#- h5py==3.7.0
+#- hdf4==4.2.15
+#- hdf5==1.12.2
+#- intake==0.6.5
+#- intake-esm==2021.8.17
+#- intake-xarray==0.6.0
+- ipycanvas==0.12.1
+- ipydatagrid==1.1.12
+- ipympl==0.9.2
+- ipyparallel==8.4.1
+- jupyter-book==0.13.1
+- jupyter-repo2docker==2022.2.0
+- jupyter-resource-usage==0.6.1
+- jupyter_bokeh==3.0.4
+- jupyterlab==3.4.5
+- jupyterlab-drawio==0.9.0
+- jupyterlab-favorites==3.0.0
+- jupyterlab-geojson==3.2.0
+- jupyterlab-git==0.39.0
+- jupyterlab-link-share==0.2.4
+- jupyterlab-variableinspector==3.0.9
+- jupyterlab_pygments==0.2.2
+- jupyterlab_widgets==3.0.2
+- jupytext==1.14.0
+- matplotlib==3.5.3
+- matplotlib-inline==0.1.6
+- mock==4.0.3
+- nbdime==3.1.1
+#- networkx==2.8.6
+- numba
+- numpy==1.23.2
+- pandas==1.4.3
+- pandoc==2.19.2
+- pandocfilters==1.5.0
+#- pep8==1.7.1
+#- pillow==9.2.0
+#- plotly==5.10.0
+#- pooch==1.6.0
+#- prettytable==3.4.1
+- pyarrow==9.0.0
+#- pypdf2==2.10.4
+#- pytables==3.7.0
+#- pytest==7.1.2
+#- pytest-cov==3.0.0
+#- pytest-notebook==0.6.1
+#- python-pdfkit==1.0.0 
+- requests==2.28.1
+#- scikit-image==0.19.3
+#- scikit-learn==1.1.2
+#- scipy==1.9.0
+#- seaborn==0.11.2
+- sphinx-jupyterbook-latex==0.4.6
+#- sqlparse==0.4.3
+#- statsmodels==0.13.2
+#- sympy==1.10.1
+#- tqdm==4.64.0
+#- xarray==2022.6.0
+#- xlrd==2.0.1
+- micro==2.0.8
+- websockify
+- pip
+- pip:
+  - -r infra-requirements.txt
+  - jupyter-desktop-server
+    # For push authentication to GitHub
+#  - gh-scoped-creds==4.1
+  # Upgrade separate from what everyone else uses for now
+  # https://github.com/berkeley-dsep-infra/datahub/issues/3693
+  - otter-grader==4.0.1
+  - ipython-sql==0.4.1
```

Comment:
You probably don't need ipyparallel, pyarrow, ipython-sql, pandas, numpy, etc. unless you want to evaluate their UIs for accessibility.

Reply:
Thanks, @ryanlovett I will keep pandas and numpy as they are commonly used packages, and can test a few sample notebooks using them. Rest of the packages - I will remove them in the upcoming commit.